docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,212 @@
+
+- **MongoDB Oplog Details**
+- **MongoDB Collection Overview**
+- **Replica Set Summary**
+- **Sharded Cluster Summary** (formerly Cluster Summary)
+
+#### Accessing the new dashboards
+
+You can find the updated dashboards in the **MongoDB** and **Dashboards** sections of the main menu. They replace the previous versions as the default dashboards for MongoDB monitoring.
+
+The old dashboards have been moved to the **Experimental** folder and renamed with **(Old)** appended to their names. Future releases will remove these older versions.
+
+#### Updated MongoDB menu structure
+
+To complement the dashboard improvements, we've also restructured the MongoDB section on the main menu for better navigation and accessibility:
+
+- Reorganized menu for a more intuitive navigation, prioritizing the two updated GA dashboards above.
+- Decluttered view by moving older dashboards to the **Experimental** folder or removing them from the menu.
+- Added **MongoDB Instance Summary**, **Collection Overview** and **MongoDB Oplog Details** the main menu.
+  
+    ![New MongoDB menu structure](../_images/new_Mongo_menu.png)
+
+We encourage you to start using the new dashboards to benefit from their enhanced monitoring capabilities. We also invite you to share your feedback in the [PMM forum](https://forums.percona.com/c/percona-monitoring-and-management-pmm/percona-monitoring-and-management-pmm-v2/31) so we can continue to improve them.
+
+### New MongoDB collector: CurrentOp
+
+The MongoDB exporter now includes the **CurrentOp** collector, offering visibility into active operations, including the new `mongodb_currentop_query_uptime` metric.
+
+To start the MongoDB exporter with the CurrentOp collector enabled, use the `--enable-all-collectors` flag when adding MongoDB services to your PMM instance:
+
+```sh
+pmm-admin add mongodb --enable-all-collectors ...
+```
+
+Specifying a limit with `--max-collections` for this collector is not necessary.
+
+!!! note "Operation collection limitation"
+    To minimize impact on disk usage, the CurrentOp collector is designed to collect only operations that have been running for longer than 1 minute. This limitation helps focus on potentially problematic long-running operations while keeping data volume manageable.
+
+For more information on MongoDB collectors and metrics, see the [pmm-admin commands documentation](../details/commands/pmm-admin.md).
+
+### Monitoring for Percona Backup for MongoDB
+
+PMM now includes support for monitoring your Percona Backup for MongoDB (PBM) setup, providing valuable insights with metrics related to `pbm-agent` statuses, PITR (Point-In-Time Recovery) configuration, and backup statuses.
+
+This new functionality is enabled through a dedicated PBM collector within the MongoDB exporter, allowing you to easily monitor your PBM setup with the following metrics:
+
+- `mongodb_pbm_cluster_backup_configured`
```

Comment:
Hi @catalinaadam it would be useful to add a sentence for each of this metric. This can be part of a wider effort (PMM-12180) - but at least we can start for the newly introduced metrics. In my opinion, we can add something here, or consider a new reference page for metrics, so we centralize their documentation.  For each of these new metric, if we create a new page, consider a table with at least: metric name, version introduced, description. For description e.g. something like: will return 1 if this cluster has PBM configured, 0 otherwise.  Or, will return 1 if the PBM is up an running, and zero otherwise (etc, depending on the specific case).  Hopefully @idoqo can help with the specific meaning of the new metrics. This will help alot both people internally and externally. Thanks.